chapter11/ard/ex2.pl

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/perl
+use strict;
+use warnings;
+
+use Oogaboogoo::Date qw( day_name month_name );
+use Time::Piece;
+
+# you can't make me use localtime!
+my $t = new Time::Piece;
+
+printf( "Today is %s %s %s %s\n", day_name( $t->wday ), month_name( $t->mon ), $t->mday, $t->year );
```

Comment:
I just got lazy - it was too late at night and I'd had wine!  Wonder if I can start using that on CRPs...